### PR TITLE
feat: update readme

### PR DESCRIPTION
**Pull request by sailorlqh**
fixes: #175

### CODE REVIEW

File: README.md
Comment (capri-xiyue) [inline README.md:33]: As I've mentioned, I will handle the user facing part including user workflow and set up, you can see https://github.com/abcxyz/pmap/pull/178

**Review by capri-xiyue — CHANGES_REQUESTED** [review]

**Comment by sailorlqh** [comment]
/Hold 
Seems like this PR has conflict with #178, I will update until that PR is merged.

File: README.md
Comment (capri-xiyue) [inline README.md:15]: 1. Pmap includes two parts, both resource mapping and wipeout plan. The architecture looks like it only covers resource mapping.
2. Cloud Asset Inventory is just one kind of Processors, make sure the picture shows it clearly. You can refer to the design doc architecture picture
3. For the `User Operation` part, I'd suggesting following https://github.com/abcxyz/jvs#architecture, the separation of `user operation` is not clear as it involves two parts, individual team to register the resource mappings and central privacy team to set up the github repo and github workflow. You don't have to separate `user operation` and `pmap service` here

Comment (capri-xiyue) [reply README.md:15]: In addition, usually data governor won't query BQ directly. Central privacy team will create a dashboard based on the BQ warehouse and the data governors can view the dashboard.

**Comment by capri-xiyue** [comment]
> /Hold Seems like this PR has conflict with #178, I will update until that PR is merged.

Don't need to hold it. You can focus on the architecture graph and the explanation of architecture graph, I will take care of background, set up and end user workflow.

File: README.md
Comment (sailorlqh) [reply README.md]: Updated.

File: README.md
Comment (sailorlqh) [reply README.md:9]: Updated the diagram.

File: README.md
Comment (sailorlqh) [reply README.md]: Updated.

File: README.md
Comment (capri-xiyue) [inline README.md:11]: This is not called the end user workflow.

**Comment by capri-xiyue** [comment]
To make it easier for review, I will merge this PR with https://github.com/abcxyz/pmap/pull/178. Thanks to Qinhang for drawing the architecture graph!

**Pull request closed without merging**
